libraries/slave_jnlp.rb

```diff
@@ -54,19 +54,18 @@ def load_current_resource
 
     def action_create
       super
-
-      parent_remote_fs_dir_resource.run_action(:create)
-
-      # don't create user/group on Windows
+    
```

Comment:
I'm curious - why did you move this code into the unless block.  The overriden code below seems to do the same thing as the one here.  Wouldn't just overriding remote_fs_dir_resource be enough?